src/toil/fileStores/cachingFileStore.py

```diff
@@ -1684,33 +1676,65 @@ def shutdown(cls, dir_):
                in the database.
         """
 
-        # We don't have access to a class instance, nor do we have access to
-        # the workflow attempt number that we would need in order to find the
-        # right database. So we rely on this hard link to the most recent
-        # database.
-        dbPath = os.path.join(dir_, 'cache.db')
-
-        if os.path.exists(dbPath):
-            try:
-                # The database exists, see if we can open it
-                con = sqlite3.connect(dbPath)
-            except:
-                # Probably someone deleted it.
-                pass
-            else:
-                # We got a database connection
-
-                # Create the tables if they don't exist so deletion of dead
-                # jobs won't fail.
-                cls._ensureTables(con)
+        if os.path.exists(dir_) and os.path.isdir(dir_):
```

Comment:
`os.path.isdir(dir_)` alone should be sufficient, as it will return `False` if it doesn't exist.